service/src/main/java/bio/terra/landingzone/stairway/flight/LandingZoneDefaultParameters.java

```diff
@@ -28,7 +28,7 @@ public static Map<String, String> get() {
         UUID.randomUUID().toString());
     defaultValues.put(
         CromwellBaseResourcesFactory.ParametersNames.POSTGRES_SERVER_SKU.name(),
-        "Standard_D2ds_v5");
+        "Standard_D2ds_v4");
```

Comment:
South Central US only supports versions 3 and 4 currently. https://learn.microsoft.com/en-us/azure/postgresql/flexible-server/overview